api.md

```diff
@@ -1,2716 +1,101 @@
-    },
-
-    // clientSupports (JSON object) identifies features supported by the client.
-    "clientSupports": {
-      // hairpinning (boolean) is 'true' if your router can route connections
-      // from endpoints on your LAN back to your LAN using those endpoints’
-      // globally-mapped IPv4 addresses/ports
-      "hairPinning": false,
-
-      // ipv6 (boolean) is 'true' if the device OS supports IPv6,
-      // regardless of whether IPv6 internet connectivity is available.
-      "ipv6": false,
-
-      // pcp (boolean) is 'true' if PCP port-mapping service exists on
-      // your router.
-      "pcp": false,
-
-      // pmp (boolean) is 'true' if NAT-PMP port-mapping service exists
-      // on your router.
-      "pmp": false,
-
-      // udp (boolean) is 'true' if UDP traffic is enabled on the
-      // current network; if 'false', Tailscale may be unable to make
-      // direct connections, and will rely on our DERP servers.
-      "udp": true,
-
-      // upnp (boolean) is 'true' if UPnP port-mapping service exists
-      // on your router.
-      "upnp": false
-    }
-  },
-
-  // tags (array of strings) let you assign an identity to a device that
-  // is separate from human users, and use it as part of an ACL to restrict
-  // access. Once a device is tagged, the tag is the owner of that device.
-  // A single node can have multiple tags assigned. This value is empty for
-  // external devices.
-  // Learn more about tags at https://tailscale.com/kb/1068/.
-  "tags": ["tag:golink"],
-
-  // tailnetLockError (string) indicates an issue with the tailnet lock
-  // node-key signature on this device.
-  // This field is only populated when tailnet lock is enabled.
-  "tailnetLockError": "",
-
-  // tailnetLockKey (string) is the node's tailnet lock key. Every node
-  // generates a tailnet lock key (so the value will be present) even if
-  // tailnet lock is not enabled.
-  // Learn more about tailnet lock at https://tailscale.com/kb/1226/.
-  "tailnetLockKey": "",
-
-  // postureIdentity contains extra identifiers from the device when the tailnet
-  // it is connected to has device posture identification collection enabled.
-  // If the device has not opted-in to posture identification collection, this
-  // will contain {"disabled": true}.
-  // Learn more about posture identity at https://tailscale.com/kb/1326/device-identity
-  "postureIdentity": {
-    "serialNumbers": ["CP74LFQJXM"]
-  }
-}
-```
-
-### Subnet routes
-
-Devices within a tailnet can be set up as subnet routers.
-A subnet router acts as a gateway, relaying traffic from your Tailscale network onto your physical subnet.
-Setting up subnet routers exposes routes to other devices in the tailnet.
-Learn more about [subnet routers](https://tailscale.com/kb/1019).
-
-A device can act as a subnet router if its subnet routes are both advertised and enabled.
-This is a two-step process, but the steps can occur in any order:
+**[Readme](./publicapi/readme.md)**
```

Comment:
Text of link should still be something like "Overview"

Reply:
Updated! (I'm getting this closed out for charlotte)